src/signals/incident/components/IncidentNavigation/index.js

```diff
@@ -16,41 +16,44 @@ const IncidentNavigation = ({ controls, meta: { wizard, submitting, handleSubmit
   const hideSubmit = controls?.hide_navigation_buttons?.meta ? controls.hide_navigation_buttons.meta.isVisible : false;
 
   return (
-    <WithWizard
-      render={({ next, previous, step }) => {
-        const currentStep = (step && step.id && step.id.split('/').pop()) || 0;
-        const wizardStep = currentStep && wizard[currentStep];
+    (wizard && (
+      <WithWizard
+        render={({ next, previous, step }) => {
+          const currentStep = (step && step.id && step.id.split('/').pop()) || 0;
```

Comment:
While you're at it, you could also change this line in
```jsx
const currentStep = (step?.id?.split('/').pop()) || 0;
```

Reply:
Yes!